frontend/src/style/main.styl

```diff
@@ -0,0 +1,3 @@
+$input-font-size = 12px
+
+@import '../../node_modules/vuetify/src/stylus/main'
```

Comment:
Is it a common practice to import file from node_modules?

Reply:
ya, thats only way we can override original file stylus